src/App.js

```diff
@@ -14,11 +14,103 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGoods: ['Jam'],
+  }
+
+  clearSelected = () => {
+    this.setState({ selectedGoods: [] });
+  }
+
+  addSelected = (item) => {
+    if (this.state.selectedGoods.includes(item)) {
+      return;
+    }
+
+    this.setState(({ selectedGoods }) => ({
+      selectedGoods: [...selectedGoods, item],
+    }));
+  }
+
+  deleteSelected = (item) => {
+    this.setState(state => ({
+      selectedGoods: state.selectedGoods.filter(product => product !== item),
+    }));
+  }
+
+  setTitle = () => {
+    let str = '';
+    const { selectedGoods } = this.state;
+    const arr = selectedGoods.slice(0, -1);
+
+    if (selectedGoods.length === 1) {
+      str = `${selectedGoods} is sellect`;
+    } else if (selectedGoods.length > 1) {
+      // eslint-disable-next-line
+      str += `${arr.join(', ')} and ${selectedGoods[selectedGoods.length - 1]} are selected`;
```

Comment:
```suggestion
      str += `${arr.join(', ')} ` 
        + `and ${selectedGoods[selectedGoods.length - 1]} `
        +  `are selected`;
```
And no need to disable linter